app/models/nic/base.rb

```diff
@@ -73,6 +73,10 @@ class Jail < ::Safemode::Jail
             :attached_devices_identifiers, :primary, :provision, :inheriting_mac
     end
 
+    def physical?
```

Comment:
sorry I forgot to mention this previously, could you add this to Jail above please?